provider/k8s.go

```diff
@@ -7,6 +7,7 @@ import (
 	"io"
```

Comment:
Why are we setting azure creds as env vars?
`	os.Setenv("AZURE_STORAGE_ACCOUNT", azureStoreConfig.AccountName)
	os.Setenv("AZURE_STORAGE_KEY", azureStoreConfig.AccountKey)`